jujugui/static/gui/src/app/components/charmbrowser/charmbrowser.js

```diff
@@ -30,14 +30,12 @@ YUI.add('charmbrowser-component', function() {
       charmstoreSearch: React.PropTypes.func.isRequired,
       charmstoreURL: React.PropTypes.string.isRequired,
       deployService: React.PropTypes.func.isRequired,
-      displayPlans: React.PropTypes.bool.isRequired,
```

Comment:
displayPlans was only toggled on legacy Juju - which means that it has been `true` all the time in Juju 2